finance-api/user-guides/fetch-payout-info.md

```diff
@@ -0,0 +1,53 @@
+Fetch payout information
+===
+Use the Finance API to fetch payout information from a merchant's account. The payout information includes the account balance, currency, payout, periodicity, and remaining discounts. 
+
+<!-- Is there any limit for how oldest transactions can be fetched? -->  
+
+* [Prerequisites](#prerequisites)
+* [Fetch payout information](#fetch-payout-information)
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorization is set up with the required OAuth scope using [Authorization OAuth2 API](../../authorization.adoc). 
+<!-- to be continued if any -->
+
+## Fetch payout information
+You can fetch payout information from a merchant's Zettle account for any given day.
+
+Fetch payout information on a specific day. If you don't specify `at`, you will get the latest payout information by default.
+     
+   ```
+   GET /organizations/self/payout-info?at={when_to_fetch_payout}
+   ```
+
+   Example:
+   
+   The following example fetches the payout information for 7 June, 2021. The next payout amount will be 6605.89 kr.
+   
+   Request   
+   ```
+   GET /organizations/self/payout-info?at=2021-06-07
+   ```
+   Response   
+   ```json
+      {
+           "data": {
+               "totalBalance": 660589,
+               "currencyId": "SEK",
+               "nextPayoutAmount": 660589,
+               "discountRemaining": 0,
+               "periodicity": "DAILY"
```

Comment:
There seems to different currencies used in different example.  `GBP` was used for this example `/organizations/self/accounts/preliminary/balance`

I think it's better to stick to one currency for all the examples and maybe best to use `GBP` since it's widely known internationally when compared to SEK.

Reply:
<img width="1042" alt="Screenshot 2021-07-07 at 12 27 45" src="https://user-images.githubusercontent.com/18021829/124744180-c7db8f00-df1e-11eb-8591-ba8631fc7eeb.png">

Here also SEK being used.